packages/gatsby-cli/src/reporter/loggers/ink/components/develop.js

```diff
@@ -0,0 +1,46 @@
+import React, { useContext, useState, useEffect } from "react"
+import { Box, Color, StdoutContext } from "ink"
+import { connect } from "react-redux"
+
+// Track the width and height of the terminal. Responsive app design baby!
+const useTerminalResize = () => {
+  const { stdout } = useContext(StdoutContext)
+  const [sizes, setSizes] = useState([stdout.columns, stdout.rows])
+  useEffect(() => {
+    stdout.on(`resize`, () => {
+      setSizes([stdout.columns, stdout.rows])
+    })
+    return () => {
+      stdout.off(`resize`)
+    }
+  }, [stdout])
+
+  return sizes
+}
+
+const Develop = ({ pagesCount, appName, errorCount, working, status }) => {
+  const [width] = useTerminalResize()
+
+  return (
+    <Box flexDirection="column" marginTop={2}>
+      <Box textWrap={`truncate`}>{`—`.repeat(width)}</Box>
+      <Box height={1} flexDirection="row">
+        <Color>{pagesCount} pages</Color>
+        <Box flexGrow={1} />
+        <Color>{status}</Color>
```

Comment:
TBD: do we want to keep it? If so we should convert machine values (`IN_PROGRESS`, `SUCCESS`, `FAILED`) to more human readable output

Reply:
probably human-readable but fine for now